src/protocols/pubsub/trie.c

```diff
@@ -299,7 +298,6 @@ int nn_trie_subscribe (struct nn_trie *self, const uint8_t *data, size_t size)
     memmove (ch->prefix, ch->prefix + pos + 1, ch->prefix_len);
     ch = nn_node_compact (ch);
     *nn_node_child (*node, 0) = ch;
-    pos = (*node)->prefix_len;
```

Comment:
@gdamore This looks like it may have been important at one time, but it's dead code now. I've tried to guess its intended meaning, but can't figure out anything it should obviously be. Shall we move forward with removing it?

Asked another way, is this orphaned code that should be removed, or an unfinished thought that should be finished?